packages/sitecore-jss-dev-tools/src/templating/nextjs/generate-component-builder.ts

```diff
@@ -0,0 +1,64 @@
+import path from 'path';
+import fs from 'fs';
+import { getComponentBuilderTemplate } from './templates/component-builder';
+import { PackageDefinition, getComponentList } from '../components';
+import { watchItems } from '../utils';
+
+// Default destination path for component builder
+const componentBuilderOutputPath = 'src/temp/componentBuilder.ts';
+const defaultComponentRootPath = 'src/components';
+
+/**
+ * Generate component builder based on provided settings
+ * @param {Object} settings settings for component builder generation
+ * @param {string} settings.componentRootPath path to components root
+ * @param {PackageDefinition[]} settings.packages list of packages to include in component builder
+ * @param {boolean} settings.watch whether to watch for changes to component builder sources
+ */
+export function generateComponentBuilder({
+  componentRootPath = defaultComponentRootPath,
```

Comment:
Should we allow them to change the `componentBuilderOutputPath` as well?

Reply:
Good point, will not hurt